_docs/_latest/develop/dev-setup.md

```diff
@@ -52,64 +53,57 @@ To set up your Cloud SQL instance for Forseti, follow the steps below:
       ```bash
       $ <path/to/cloud_sql_proxy> -instances=INSTANCE_CONNECTION_NAME=tcp:3306
       ```
-      
-1. Make a note of your the Cloud SQL user you created (e.g. "forseti_user") as well as 
-   the database name (e.g. "forseti_security" -- this is NOT the ID of your Cloud SQL instance). 
-   You will need these for your forseti_conf.yaml later.
+
```

Comment:
Not here, but in the steps to "Setting up Cloud SQL" should be documented in an include so we can re-use it other places.

Reply:
Did this get resolved?